docs/project/contributing.md

```diff
@@ -36,9 +54,17 @@ https://github.com/herbsjs
 ```
 2. Click on the Issues tab
 
-Just choose which question you want to act on and comment that you will act so other members are aware and your effort is not wasted.
+Just choose which issue you want to act on and comment that you will act so other members are aware and your effort is not wasted.
 You will have access to all open issues and also those that have already been closed.
 
+Below we have some links to make your search easier if you want to know all the open PRs without anyone acting or want to give tips or even help someone in an issue that is under development.
+
```

Comment:
Please add a Header before this paragraph, like: "Want to know how to start?" or something simular